client/schema.go

```diff
@@ -0,0 +1,37 @@
+package client
+
+import (
+	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/strutil"
+)
+
+// SupportedSchema returns a slice of different OpenLDAP schemas supported
+// by the plugin.  This is used to change the FieldRegistry when modifying
+// user passwords.
+func SupportedSchema() []string {
```

Comment:
```suggestion
func SupportedSchemas() []string {
```

Would be more accurate.

Reply:
Pushed this in https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/5/commits/6008ee92d0209ad00eabaae4018940c2be03d2da